### ISSUE TITLE
Sleep Mode

### ISSUE THREAD

**Issue by zHostingSolutions** [zpanel/zpanelx#171]
A Sleep Mode core module would be great in order to put the server to sleep at any time from the control panel. Also, if server admin wants to set a time for server to automatically go to sleep, a time can be set and it will automatically go to sleep at that time if the server is awake.

This way, since, the computer is technically still on, but, running at low power, then zPanel and domains will still run without the computer running hot which will keep the computer working longer due to not being heated all the time, or over heated.

**Comment by trojanspike** [comment]
Yea , you should make it if it sounds good to you.

It's all well and good coming up with ideas and waiting for others to create it, but it's very unlikely someone is going to go out of their way to make it unless there is a very good need for it and by you making it it'll give you some experience in the workings of Zpanel.

**Comment by ghost** [comment]
This is a joke a sleep mode in a server. are we talking here about production server available 24/7?
Server overheating? In DC that's never an issue buddy.

Guys this space would be used mainly for BUGS if you want suggest features use the forums to discuss it.

@bobsta63  you can close this thread

Me.B

**Comment by zHostingSolutions** [comment]
@trojanspike I have only limited, basic php knowledge and don't know how to create any of the modules that I request, especially, when it comes to affecting they computer system. This is why I put a request for the experts do it.

I've said on the forum before, that I have limited knowledge. I also said plenty of times that when I request ANY module, I KNOW everyone it busy. Also, I'm NEVER asking for myself, EVERYONE would be able to use and enjoy them. I know modules, updates and server rewrites take time.

If I knew how to do it, I WOULD create the modules that I request AND make it available for EVERYONE to use. I also said that, too. No one here ever has to worry about me selfishly making modules for me, personally. It's for EVERYONE.

I will continue to use the forum to make requests, though. I didn't know that things changed when zpanel started using github.
